certificate-policy.md

```diff
@@ -1250,26 +1250,24 @@ No stipulation.
 
 #### 6.5.1 Specific computer security technical requirements
 
-Administrator privileges to all Certificate System Components shall only be granted to the Administrator trusted role.  All CAs shall implement multi-factor or multi-party authentication for all Administrator trusted role access to Certificate System Components including operating system and software.
-
-All CAs shall implement multi-factor authentication for the Officer trusted role.
-
 For all CAs and Certificate System Components including certificate status services, the computer security functions listed below are required. These functions may be provided by the operating system, or through a combination of operating system, software, and physical safeguards. The Certificate System Components shall include the following functionality:  
 
 - Be configured to remove or disable all accounts, applications, services, protocols, and ports that are not used in the CA's operations
 - Authenticate the identity of users before permitting access to the system or applications
-- Manage privileges of users to limit users to their assigned roles and implement least privilege controls  
+- Manage privileges of users to limit users to their assigned roles and implement least privilege controls
+- Enforce separation of duties for roles
 - Generate and archive audit records for all transactions (see Section 5.4)  
 - Enforce domain integrity boundaries for security critical processes
+- Provide process isolation
 - Support recovery from key or system failure
 
-For remote workstations used to administer the CAs, the computer security functions listed below are required:
+When CA and Certificate System Components including certificate status services, are hosted on evaluated platforms in support of computer security assurance requirements then the system (hardware, software, and operating system) shall, when possible, operate in an evaluated configuration.  At a minimum, such platforms shall use the same version of the computer operating system as received the evaluation rating.
```

Comment:
sticking point: 

- Evaluated platforms is not defined and open to interpretation (similar to our "authorized entity" discussion). Can we define evaluated with a more discrete definition that isn't open to interpretation for an auditor?